public/app/percona/settings/Settings.messages.ts

```diff
@@ -48,6 +48,10 @@ export const Messages = {
     technicalPreviewLinkText: 'here',
   },
   alertmanager: {
+    warningPre: "Note: integration with Alertmanager is needed only in cases when you can't use",
+    warningLinkContent: 'Integrated Alerting',
+    warningLinkURL: '/graph/integrated-alerting',
```

Comment:
Can this link go to the `Settings.constants` file instead of messages file?

Reply:
yeah, for sure :))